deployment.go

```diff
@@ -47,7 +47,7 @@ type DeploymentStep struct {
 }
 
 // DeploymentStepV1 is a Marathon v1.X-compatible DeploymentStep
```

Comment:
The name in the comment needs to be updated. I'd also suggest a more generic descriptions, such as:

_StepActions is a series of deployment steps._